src/views/HomeView.vue

```diff
@@ -1,6 +1,17 @@
 <template>
   <main class="page-content">
     <div class="container">
+      <div :class="$style.dealsList">
+        <DealCard
+          v-for="(dealCard, index) in dealsData"
+          :key="`dealCard__${index}`"
+          :variant="dealCard.variant"
+          :title="dealCard.title"
+          :offer="dealCard.offer"
+          :tag="dealCard.tag"
+        />
+      </div>
+
       <BaseAlert variant="primary" :message="`Email: ${formData.email}`" />
```

Comment:
Please remove unnecessary code in Home page, leave just what you need in home page

Reply:
I tried it. but if I turn it off, the data disappears